packages/typechain-target-fuels/src/index.ts

```diff
@@ -3,12 +3,14 @@ import { readFileSync } from 'fs';
 import { join, resolve } from 'path';
 import type { Dictionary } from 'ts-essentials';
 import type { CodegenConfig, Config, FileDescription } from 'typechain';
-import { getFilename, normalizeName, TypeChainTarget } from 'typechain';
+import { getFilename, TypeChainTarget } from 'typechain';
+
 
 import { codegenAbstractContractFactory, codegenContractTypings } from './codegen';
 import { FACTORY_POSTFIX } from './common';
 import { extractAbi, extractDocumentation, parse } from './parser/abiParser';
 import type { Contract, RawAbiDefinition } from './parser/abiParser';
+import { normalizeName } from './parser/parseSvmTypes';
```

Comment:
Running it locally here; Looks like it solved the class names but not file names, resulting in issues with imports.

Reply:
Yeah, I think the issue is https://github.com/FuelLabs/fuels-ts/blob/master/packages/typechain-target-fuels/src/parser/abiParser.ts#L191